src/components/PageHeader.astro

```diff
@@ -64,7 +67,7 @@ const isHelp = uid?.split("-")[0] == "help";
         )
       }
       {
-        !isLearn && (<Share description={description} title={title} />
+        !(isLearn || isWebApi) && (<Share description={description} title={title} />

```

Comment:
This type of dependency is what I worry about. All pages 'should' have a share component. The only page types that do not (should not) today are the landing and category pages.